tutorial/sample-problems/Parametric/PolarGraph.pg

```diff
@@ -55,7 +55,7 @@ BEGIN_PGML
 Find the area enclosed by one petal of the rose curve
 [`r = f(\theta) = \cos(5\theta)`].
 
->>[!a rose curve!]{$graph}{300}<<  
+>>[!a rose curve!]{$graph}{300}<<
```

Comment:
Put this line back to the way that it was.  It was that way for a reason.  The two spaces at the end of the line give a PGML break.  That makes `Graph of ...` centered on the next line instead of on the same line to the right of the image which looks quite bad.